core/src/scss/utilities/placeholders/components/_card.scss

```diff
@@ -35,20 +36,22 @@
     }
 
     > h3,
-    > h4 {
+    > span {
       @include margin(null null 0);
       @include sans;
     }
 
     // Super Headline.
-    > h4 {
-      @include modular-spacing('padding', null null -2);
+    > span {
+      @include modular-spacing('margin', null null -2);
+      @include types;
```

Comment:
You're setting `margin-bottom` on this `span` 3 times: line 40, line 46, and in the `types` mixin on line 47. It's hard to figures out what the `margin-bottom` is supposed to be.

Reply:
The `margin-bottom: 0` was odd - I'm not even sure how it got there so I got rid of it. As for the @types - that's `margin-top: 0` and all heading styles is supposed to have that. Even though the superheadline is a span, it's supposed to take on the h6 style. It's probably a good idea since it might inherit some margin-top: from the site's styles.

Reply:
Removed redundant margin-bottom